cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -85,7 +85,8 @@
       cronjob at:'30 14 * * *', do:dashboard_dir('bin','scheduled_pd_workshop_emails')
       cronjob at:'* 14 * * 1', do:dashboard_dir('bin','scheduled_pd_application_emails')
       cronjob at:'10 */12 * * *', do:dashboard_dir('bin','refresh_pd_workshop_material_orders')
-      cronjob at:'*/5 * * * *', do:dashboard_dir('bin', 'sync_jotforms')
+      cronjob at:'*/1 * * * *', do:dashboard_dir('bin', 'fill_jotform_placeholders')
+      cronjob at:'30 0 * * *', do:dashboard_dir('bin', 'sync_jotforms')
```

Comment:
Just confirming - this will sync jotform once an hour, 30 seconds in

Reply:
Line 88 is [every minute](https://crontab.guru/#*/1_*_*_*_*).
Line 89 is once a day at [00:30](https://crontab.guru/#30_0_*_*_*).